bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Carousel.java

```diff
@@ -46,6 +46,13 @@
      */
     String PN_AUTOPAUSE_DISABLED = "autopauseDisabled";
 
+    /**
+     * Name of the policy property that defines whether the control elements should be placed in front of the carousel items.
+     *
+     * @since com.adobe.cq.wcm.core.components.models 12.20.0
+     */
+    String PN_PREPEND_CONTROLS = "prependControls";
```

Comment:
I'd rename it: PN_CONTROLS_PREPENDED = controlsPredended

Reply:
adressed in a407f8573ac5905bd5d9573fdc05fca0a2753c36